src/mon/PGMap.cc

```diff
@@ -831,8 +831,7 @@ void PGMapDigest::dump_pool_stats_full(
     f->close_section();
   else {
     ceph_assert(ss != nullptr);
-    *ss << "POOLS:\n";
-    tbl.set_indent(4);
+    *ss << "--- POOLS ---\n";
```

Comment:
i think a better way is to be explicit on the required size of horizontal real estate if it really matters:
```c++
#include <fmt/ostream.h>
...
fmt::print(*ss, "{:^13}\n", "POOLS");
```

see https://fmt.dev/latest/syntax.html#format-specification-mini-language for libfmt's format specification.

Reply:
The horizontal size of the header (~20 chars?) doesn't matter.. I just want to avoid indenting the whole table (80+ chars), but depends on which columns are visible, how big the numbers are, etc.

Reply:
@tchaikov WDYT? IMO the fmt::print part just makes it harder to tell what is being printed.

Reply:
@liewegas in that case, yeah, in that case, printf() sucks in readablity. if the exact spacing is not important, and what we want is just to ditch the "set_indent()" call.